geometry_msgs/Point initial_point
geometry_msgs/Point landing_point
float64 period
float64 height
int32 resolution
---
geometry_msgs/Point[] points
builtin_interfaces/Duration[] time_from_start